rllib/models/torch/torch_distributions.py

```diff
@@ -141,7 +143,7 @@ def required_input_dim(space: gym.Space, **kwargs) -> int:
     def rsample(self, sample_shape=()):
         if self._one_hot is None:
             self._one_hot = torch.distributions.one_hot_categorical.OneHotCategorical(
-                logits=self.logits, probs=self.probs
+                logits=self.logits, probs=self.probs, validate_args=False
```

Comment:
Interesting. So validation takes a long time? Shall we by time introduce a general configuration argument `validate_args` that includes validations in all such methods, like here and in gymnasium, etc.? Such it helps users in finding bugs?

Reply:
Maybe we should, yeah. Then again, don't want to add too much second-layer/wrapped config args.

I think the ultimate goal should be to try and use `torch.distributions` directly (in custom and default RLModules) without the RLlib wrappers.